I0517 04:05:07.283951 140144601069376 run.py:28] Device: cuda
I0517 04:05:07.818734 140144601069376 diagnostics.py:438] 17/769 coordinates fall outside of the mask. Removing them.
I0517 04:05:08.583692 140144601069376 diagnostics.py:438] 47/1722 coordinates fall outside of the mask. Removing them.
I0517 04:05:09.013922 140144601069376 diagnostics.py:438] 32/698 coordinates fall outside of the mask. Removing them.
I0517 04:05:10.102289 140144601069376 diagnostics.py:438] 44/2553 coordinates fall outside of the mask. Removing them.
/well/nichols/users/pra123/anaconda3/envs/torch/lib/python3.11/site-packages/patsy/util.py:672: FutureWarning: is_categorical_dtype is deprecated and will be removed in a future version. Use isinstance(dtype, CategoricalDtype) instead
  return _pandas_is_categorical_dtype(dt)
/well/nichols/users/pra123/anaconda3/envs/torch/lib/python3.11/site-packages/patsy/util.py:672: FutureWarning: is_categorical_dtype is deprecated and will be removed in a future version. Use isinstance(dtype, CategoricalDtype) instead
  return _pandas_is_categorical_dtype(dt)
